src/create-config.ts

```diff
@@ -205,32 +201,33 @@ const main = async () => {
   });
 
   const airkeeperSubscriptions = Object.values(apiData.beacons)
-    .flatMap((beacon) => {
-      const DapiServerInteface = DapiServerInterface();
-      const parameters = '0x';
-      const airnodeAddress = beacon.airnodeAddress;
-      const templateId = beacon.templateId;
-
-      const threshold = ethers.BigNumber.from(100000000)
-        .mul(beacon.updateConditionPercentage * 100)
-        .div(10000);
-      const beaconUpdateSubscriptionConditionParameters = ethers.utils.defaultAbiCoder.encode(['uint256'], [threshold]);
-      const encodedBeaconUpdateSubscriptionConditions = encode([
-        {
-          type: 'bytes32',
-          name: '_conditionFunctionId',
-          value: ethers.utils.defaultAbiCoder.encode(
-            ['bytes4'],
-            [DapiServerInteface.getSighash('conditionPspBeaconUpdate')]
-          ),
-        },
-        { type: 'bytes', name: '_conditionParameters', value: beaconUpdateSubscriptionConditionParameters },
-      ]);
-
-      return beacon.chains.map((chain) => {
-        const chainId = chainNameToChainId[chain.name] || 1;
-        //TODO: Add DapiServer contracts based on chain
-        const DapiServerAddress = ethers.constants.AddressZero;
+    .flatMap((beacon) =>
+      Object.entries(beacon.chains).map(([chainName, chain]) => {
+        const chainId = chainNameToChainId[chainName] || 1;
+        const DapiServerInteface = DapiServerInterface();
+        const parameters = '0x';
+        const airnodeAddress = beacon.airnodeAddress;
+        const templateId = beacon.templateId;
+
+        const threshold = ethers.BigNumber.from(100000000)
+          .mul(chain.updateConditionPercentage * 100)
+          .div(10000);
+        const beaconUpdateSubscriptionConditionParameters = ethers.utils.defaultAbiCoder.encode(
+          ['uint256'],
+          [threshold]
+        );
+        const encodedBeaconUpdateSubscriptionConditions = encode([
+          {
+            type: 'bytes32',
+            name: '_conditionFunctionId',
+            value: ethers.utils.defaultAbiCoder.encode(
+              ['bytes4'],
+              [DapiServerInteface.getSighash('conditionPspBeaconUpdate')]
+            ),
+          },
+          { type: 'bytes', name: '_conditionParameters', value: beaconUpdateSubscriptionConditionParameters },
+        ]);
+        const DapiServerAddress = operationsRepository.documentation.chains[chainName].contracts.DapiServer;
```

Comment:
Same here, shouldn't this come out of `/chains`? `documentation` is a secondary payload for docs use.